### PR TITLE
Exit Interview

### PR DESCRIPTION
**Pull request by elliothursh**
## Describe your changes
Exit Interview Email

### CODE REVIEW

**Comment by monicakogler** [comment]
fabulous. a few little things when you have a chance plz –

- [x] spacing btwn logo and headline should be 72. i think you can lose the 16px top padding from the headline?
- [x] in the light blue box, possible to get the line to break after "you" instead of "need" somehow?
- [x] 24 instead of 32 above gray initial loan amount box 
- [x] possible to get 4px border radius on the white boxes inside the payment reduction and pause sections?
- [x] can callout-right-col be 158px wide? this gets slightly nicer line breaks

**Comment by alexfarrill** [comment]
The logo and headline spacing might be due to there being no icon below the Fynn logo on this one, other templates have an icon in that space

**Comment by alexfarrill** [comment]
lgtm on Superhuman desktop & mobile and in gmail desktop

**Comment by elliothursh** [comment]
updated!

**Comment by monicakogler** [comment]
cool, the changes lgtm, thanks. but the images aren't loading?

**Comment by elliothursh** [comment]
ahh, i messed up the url. its fixed now 👍

**Review by psugihara — APPROVED** [review]
👨‍🍳 💋

File: fynn/emails/ExitInterview.tsx [hunk 1]
```diff
@@ -0,0 +1,734 @@
...
+              <MjmlImage
+                href="https://fynncredit.com"
+                src={assetUrl("/assets/logo-full.png")}
+                alt="Fynn Logo"
+                height="28px"
+                width="63px"
+                paddingTop="0px"
+                align={"left"}
+              />
+              <Spacer lg={{ height: 64 }} sm={{ height: 32 }} />
```
Comment (monicakogler) [inline fynn/emails/ExitInterview.tsx]: can this be 48 instead of 64?

**Review by monicakogler — APPROVED** [review]

**Pull request merged**